core/src/main/java/com/oath/oak/InternalOakMap.java

```diff
@@ -805,56 +835,47 @@ boolean computeIfPresent(K key, Consumer<OakWriteBuffer> computer) {
             throw new NullPointerException();
         }
 
-        int infiLoopCount = 0;
-        while (true) {
-            infiLoopCount++;
-            if (infiLoopCount > 1000) {
-                System.out.println("Stuck in non-zc get");
-                assert false;
-            }
+        ThreadContext ctx = getThreadLocalContext();
```

Comment:
Why not to get context as input parameter? It exists just above. The code for finding this thread context runs for nothing... Also in all getXTransformation() methods below....